docs/lumen-installation.md

```diff
@@ -54,3 +54,39 @@ This will update your `.env` file with something like `JWT_SECRET=foobar`
 
 It is the key that will be used to sign your tokens. How that happens exactly will depend
 on the algorithm that you choose to use.
+
+### Generate certificate
```

Comment:
The instruction are identical to the `laravel-installation.md`, aren't they?

Can we make a dedicated section for this somewhere, without duplicating this?

Reply:
I have changed this in https://github.com/PHP-Open-Source-Saver/jwt-auth/pull/146